adal/versioning/version.properties

```diff
@@ -1,3 +1,3 @@
 #Wed Aug 01 15:24:11 PDT 2018
-versionName=3.1.2
+versionName=3.1.3-RC1
```

Comment:
This also still has the RC

Reply:
Yes, though I performed the merge against origin/release/3.1.3, it still pulled from my out of date local branch.  I subsequently fixed it.

Reply:
And evidently not completely.